imports/email-templates/case-new-message.js

```diff
@@ -1,24 +1,24 @@
 import url from 'url'
-import { optOutHtml, optOutText } from './components/helpers'
+import { address, optOutHtml, optOutText } from './components/helpers'
 
 export default (assignee, notificationId, settingType, caseTitle, caseId, userId, message) => ({
```

Comment:
If `userId` is not the user's id, then it should be named differently.
Just `user`?